ui/src/app/views/project/add/project.add.html

```diff
@@ -47,6 +47,13 @@
                     <app-permission-form (createGroupPermissionEvent)="permissionManagement($event)" buttonType="button" #permForm></app-permission-form>
                 </div>
             </div>
+            <div class="row" *ngIf="(!project.groups || project.groups.length <= 0) && groupError">
+                <div class="column">
+                    <div class="ui error message">
+                        {{ 'project_groups_error' | translate }}
+                    </div>
+                </div>
+            </div>
```

Comment:
In fact the error position change if there are group or not  
No group
![image](https://cloud.githubusercontent.com/assets/1301995/26285434/135a2bae-3e50-11e7-96b2-2c39e09cc905.png)
Groups but with other errors
![image](https://cloud.githubusercontent.com/assets/1301995/26285439/26c472c6-3e50-11e7-9bc9-7f76f4237a08.png)
But i can remove the first div and keep only the new